setup.py

```diff
@@ -41,6 +41,7 @@
         'plone.app.z3cform',
         'plone.behavior',
         'plone.dexterity',
+        'plone.api',
```

Comment:
plone.api can not be used on core packages, as sad as it is that's the decision so far... maybe the @plone/framework-team wants to revisit this at some point :-)